.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 on: [push]
 jobs:
   python-tests:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Might be best to keep this at the same version as the heroku stack (currently 20) to avoid any surprises - https://github.com/mitodl/mitxpro/issues/2496

```
-----> Building on the Heroku-20 stack
```